bin/posthog-plugin-server

```diff
@@ -1,3 +1,3 @@
 #!/usr/bin/env node
 
-require(__dirname + '/../dist/index.js')
+require(__dirname + '/../dist/src/index.js')
```

Comment:
This `/dist/src/` structure looks a bit odd, can we avoid the `src` here?

Reply:
I agree amd I tried to... yet wasting enough effort trying, without some manual rm/mv/etc steps, I couldn't get this to work :(.

The problem was mainly with `package.json`, which is in the root folder and thus the typescript wants to move the root of "src" also into dist.

Solving that, the next problem was with `allowJs`. We need that for the `worker/piscina.js` file, which runs the workers as needed. Changing that to ".ts" and having it work with all "ts-node" (start:dev), "tsc" (build) and "jest" (test) seemed like a bigger rabbit hole that I was willing to get into. 

Thus to not waste too much time here, I just moved on :/.

Reply:
Hm, okay, maybe some other time.